galaxy_ml/keras_galaxy_models.py

```diff
@@ -1104,7 +1109,12 @@ def fit(self, X, y=None, class_weight=None, sample_weight=None, **kwargs):
         self.data_generator_.set_processing_attrs()
 
         if y is not None:
-            X, y = check_X_y(X, y, accept_sparse=['csr', 'csc'], allow_nd=True)
+            multi_output = False
```

Comment:
@qiagu This is the change I pushed. Added the fix to the other method too, as you suggested.

Reply:
Okay. Thanks.